### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by CSchoel**
During my work on #358, I noticed a few parts of the README_detailed.md that could be improved in my opinion. I didn't want to conflate this with the code and page changes, so I created a separate branch.

When I first read the README, I missed the part that mentioned the `Makefile`, so I thought it would make sense to make this more prominent and give a TLDR version for the hasty reader. :wink: 

In the more detailed description afterward, there was some confusion about where generated files would end up. I guess this has changed since the README was written, but wasn't updated here.

Finally, I was confused that not all the required files were in the `build` folder after generation, so I added the info that you need to copy the files from `hugo` into `build` before running Hugo.

### CODE REVIEW

**Review by mbollmann — APPROVED** [review]
Thanks, that looks good to me!

I wonder to what extent we need to describe the "manual" build steps at all. In principle, everything should be doable through the Makefile, as the individual build steps should all have their own make target anyway. I guess that's another thing we could make clearer in the future.

**Review by akoehn — COMMENTED** [review]
Forgot to submit my comments ...

File: README_detailed.md [hunk 1]
```diff
@@ -70,24 +80,29 @@ the Anthology.  This is achieved by calling:
...
 
 This creates a corresponding `.xml` file in MODS format for every `.bib` file
 generated previously.
 
 ### Step 4: Run Hugo
 
-After all necessary files have been created, the website can be built by simply
-invoking Hugo from the `build/` subdirectory.  Optionally, the `--minify` flag
-can be used to create minified HTML output:
+The files that were generated so far are in the `build/` subdirectory, in which
```
[the remark this answers is not in the document]
Comment (mjpost) [reply README_detailed.md:98]: That might be be good to do, but in the meantime this improves things, so I'm just going to merge it.

**Review by mjpost — APPROVED** [review]
Thanks for the edits!

**Pull request merged**